packages/material-ui/src/Rating/Rating.js

```diff
@@ -391,12 +482,12 @@ const Rating = React.forwardRef(function Rating(props, ref) {
         if (precision < 1) {
           const items = Array.from(new Array(1 / precision));
           return (
-            <span
+            <RatingIcon
```

Comment:
Why is this `RatingIcon`? This seems like it should be `RatinDecimal` component, right?